### ISSUE TITLE
Remove old rate limits config after new rate limit is deployed

### ISSUE THREAD

**Issue by fungc-io** [authgear/authgear-server#3098]

**Comment by louischan-oursky** [comment]
We can schedule this task now.

**Comment by fungc-io** [comment]
same as #3066?

**Comment by louischan-oursky** [comment]
No. This issue is about removing the deprecated configuration fields from the configuration file schema. After this change, any configuration file still containing the removed fields will be considered as invalid.

**Comment by tung2744** [comment]
We will run the migration script again after this cooldown, and observe if any invalid configs exist in next cooldown.
- [ ] Staging
- [ ] Production

**Comment by tung2744** [comment]
Checked in staging and there are still deprecated configs, need to check again where it comes from..
